tests/sanity/tests/planning/plan.spec.ts

```diff
@@ -377,6 +377,7 @@ test.describe('Planning ToDo tests', () => {
     })
 
     await test.step('Check time changes', async () => {
+      test.skip()
```

Comment:
I think you skipped not a test but a step. Should we skip test instead?

Reply:
:) Fixed